.github/pull_request_template.md

```diff
@@ -1,23 +1,23 @@
-**Thank you for submitting this pull request**
+**JIRA**: [link](https://www.example.com) 
 
-**JIRA**: _(please edit the JIRA link if it exists)_ 
+**Referenced Pull Requests** (if any):
 
-[link](https://www.example.com)
+**Checklist**:
 
-**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_
+- [ ] Documentation updated if applicable
+- [ ] Upgrade recipe provided if applicable
 
-paste the link(s) from GitHub here
+**Thank you for submitting this pull request!**
 
-**How to retest or run**:
+---
+<pre>
```

Comment:
Is this intentional? I don't really see a reason for the entire following text to be preformatted.
The commands themselves, yes. But everything?

Reply:
Suggestion: use `<details>`:
<details>
<summary>How to retest</summary>

- PR build
- FDB
- etc
</details>

See [Using `<details>` in GitHub](https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d).